front/app/modules/commercial/content_builder/admin/components/ContentBuilderToggle/ContentBuilderToggle.test.tsx

```diff
@@ -1,13 +1,30 @@
+import { addContentBuilderLayout } from 'modules/commercial/content_builder/services/contentBuilder';
 import React from 'react';
 import { Multiloc } from 'typings';
 import { render, screen } from 'utils/testUtils/rtl';
 import ContentBuilderToggle from './';
 
+const DEFAULT_CONTENT_BUILDER_LAYOUT_DATA = {
+  data: {
+    attributes: {
+      enabled: false,
+    },
+  },
+};
+
+let mockContentBuilderLayoutData: any = DEFAULT_CONTENT_BUILDER_LAYOUT_DATA;
```

Comment:
Let's change the type here to `typeof DEFAULT_CONTENT_BUILDER_LAYOUT_DATA | Error`. Generally, not a good idea to use `any`

Reply:
Hmm.. my tests fail if I change the type. The toggle doesn't seem to render.